confluence-syntax-xhtml/src/test/resources/confluence+xhtml10/specific/preformat/preformat2.test

```diff
@@ -0,0 +1,27 @@
+.#-----------------------------------------------------
+.input|confluence+xhtml/1.0
+.# Make sure any number of white spaces is ignored between non inline elements
+.#-----------------------------------------------------
+<pre>Some text <a href="https://domain.com/path/to/resource">link text</a> other text</pre>
+.#-----------------------------------------------------
+.expect|event/1.0
+.#-----------------------------------------------------
+beginDocument
+beginGroup [[class]=[code]]
+beginParagraph
+onWord [Some]
+onSpace
+onWord [text]
+onSpace
+beginLink [Typed = [true] Type = [url] Reference = [https://domain.com/path/to/resource]] [false] [[shape]=[rect]]
```

Comment:
Where does this shape attribute come from? But it's probably unrelated to this PR.